insonmnia/miner/config.go

```diff
@@ -4,17 +4,21 @@ import (
 	"github.com/jinzhu/configor"
 )
 
-type MinerConfig struct {
```

Comment:
Changes in this file are mostly out of the scope (and is done in https://github.com/sonm-io/core/pull/40), except for adding limitations option.

Also may be rename to `restrictions`?